src/api/store/graph.py

```diff
@@ -407,39 +407,42 @@ def list_graphs_for_super_admin(self, context: Context):
       return None, CustomMassenergizeError(str(e))
 
   def debug_data_fix(self) -> None:
-    # attempting to fix the problem with data getting screwed up
-    for community in Community.objects.all().select_related('goal'):
-      if community.goal:
-        action_goal= max(community.goal.target_number_of_actions, 100)
-      else:
-        # communities that don't have goals
-        action_goal = 100
-
-      if community.is_geographically_focused:
-        user_actions = UserActionRel.objects.filter(
-          real_estate_unit__community=community, status="DONE"
-        )
-      else:
```

Comment:
this function was breaking whenever migrations hadn't been done.  I surrounded it with a try catch.